### ISSUE TITLE
Export creator information

### ISSUE THREAD

**Issue by tlotze** [collective/collective.exportimport#38]
Currently, content is exported without creator information. There are cases, however, when that information is needed and it is not sufficient to have imported objects created by the importing user. Maybe it would be useful to export creator information optionally, but if so, I'm not sure about whether it should be done by default.

**Comment by fredvd** [comment]
I just checked one of my previous migrations because I can't remember having issues with the Creator field.  In my migration so far with collective.exportimport the creator has been copied. Maybe this is done with the local roles import where the 'Owner' local role is applied to the new content?

**Comment by pbauer** [comment]
I also checked and the creator-data is exported fine (AT and DX) . The probem you encounter is probably that there is a empty string in the list of creators. 

To fix that simply use 
```python
item["creators"] = [i for i in item.get("creators", []) if i]
```
in your `global_dict_hook`.

**Comment by mauritsvanrees** [comment]
I have the same problem. Creators is something like `['Joe', '', 'Jane']` it fails. Philip's suggestion helps.

This would be useful to add in the package. Where would we do this? Maybe do this cleanup during export, when "Modify exported data for migrations" is checked?

**Comment by Rudd-O** [comment]
Yeah I think the package should just do this.  An empty string for Creator is outright bad data.

**Comment by holdenhao** [comment]
> Currently, content is exported without creator information. 

I have experienced this as well but only for DX content that does not implement plone.ownership behavior and does not have a creators field defined in its schema. I am using c.exportimport 1.4 on Plone 5.2.7
